ext_localconf.php

```diff
@@ -3,6 +3,17 @@
 	die ('Access denied.');
 }
 
+$GLOBALS['TYPO3_CONF_VARS']['EXTCONF'][$_EXTKEY]['setup'] = unserialize($_EXTCONF);
```

Comment:
Can you try to use the API to get these values? With the benefit to get the default values even if the User hasn't saved the settings in the EM.

	/** @var \TYPO3\CMS\Extensionmanager\Utility\ConfigurationUtility $configurationUtility */
	$configurationUtility = $objectManager->get('TYPO3\CMS\Extensionmanager\Utility\ConfigurationUtility');
	$configuration = $configurationUtility->getCurrentConfiguration('vidi');

	$configuration['autoload_typoscript']['value']